templates/storage.template.yaml

```diff
@@ -172,6 +172,10 @@ Resources:
                   - 'storagegateway:DeleteFileShare'
                   - 'storagegateway:AddTagsToResource'
                 Resource: !Sub 'arn:${AWS::Partition}:storagegateway:*:*:*'
+              - Effect: Allow
+                Action:
+                  - 'kms:DescribeKey'
+                Resource: !Sub 'arn:${AWS::Partition}:kms:*:*:*'
```

Comment:
Can we restrict to a specific kms resource?

Reply:
I had misread the documentation about the File Share encryption. It's default behavior is to use server-side encryption with an S3-Managed key. Changing this with another default key doesn't really add value  so we decided to remove it.